src/pages/ProfilDetail.js

```diff
@@ -0,0 +1,34 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { goBack } from 'react-router-redux';
+import './ProfilDetail.css';
+
+class EventDetail extends PureComponent {
```

Comment:
The Component name is ProfilDetail so also name the class like that ;) 

And maybe also correct the spelling error as Profil in english is Profile with an e attached at the end.